### PR TITLE
remove redundant computation in batchnorm decomposition

### PR DESCRIPTION
**Pull request by blzheng**
This pr is to fix the regression mentioned in https://github.com/pytorch/torchdynamo/issues/1988. After this pr, the inductor performance of Resnet50 training on CPU is ~0.95x that of eager.

### CODE REVIEW

**Review by jgong5 — APPROVED** [review]
LGTM. Added @Chillee to review it.

**Review by EikanWang — APPROVED** [review]
LTGM. By the way, could you please add a UT?

**Comment by desertfire** [comment]
duplicate of https://github.com/pytorch/pytorch/pull/90984, cc @ngimel

**Comment by ngimel** [comment]
Yeah #90984 has landed, it does the same thing
